### ISSUE TITLE
Can't see how to add an external playlist

### ISSUE THREAD

**Issue by tomchiverton** [ArturSierzant/OMPD#154]
I have a file call "new.m3u" in the top level folder ( e.g. $cfg['media_dir'] ) I wan to add as a play list. There are 25 files listed in it, one per line.

No matter what I put in "URL or path:" at .../favorite.php?action=editFavorite&favorite_id=20# nothing seems to happen

```
$ pwd
/exports/mp3
$ head -n 1 new.m3u
/exports/mp3/Spring_start/Powderpaint.mp3
$ grep export /home/ompd/include/config.inc.php
$cfg['media_dir']                   = '/exports/mp3/';
```

**Comment by ArturSierzant** [comment]
It's fixed in commit e892d74. 

In "URL or path:" you need to enter full path to `m3u` file, e.g.: `/exports/mp3/new.m3u`

`m3u` file must contain full path to media files, so `/exports/mp3/Spring_start/Powderpaint.mp3` is fine. It can also contain path relative to `$cfg['media_dir'] `, so in your case `Spring_start/Powderpaint.mp3` should also work.

Please let me know if this helped and thanks for reporting this issue.

**Comment by tomchiverton** [comment]
WFM, thanks !

I use this to add all newly added singles, because I can't see how to query this - is there a way ?
